ISISScientificComputing/autoreduce-frontend
# Fix scrollbar page shift

### Summary of work
Add a vertical scroll bar to the base CSS template -- stops the entire page from reflowing because of the scrollbar coming and going. 

Remove the fullstops at the end of the footer messages.

### How to test your work
Navigate to the different pages of the webapp and make sure the vertical scroll bar is always present.
